charts/linkerd2/values.yaml

```diff
@@ -141,7 +141,7 @@ identity:
     # control plane annotation - do not edit
     crtExpiryAnnotation: linkerd.io/identity-issuer-expiry
 
-    issuanceLifetime: 86400s
+    issuanceLifetime: 24h0m0s
```

Comment:
This is how go serializes this duration.  By formatting it this way in the chart values.yaml, we avoid spuriously detecting it as overridden.